automated-comments/java/hamming/must_use_constructor.md

```diff
@@ -0,0 +1,8 @@
+This exercise must use a constructor that takes two Strings:
+```java
+Hamming(String leftStrand, String rightStrand) {
+    ...
+}
+```
+
+NOTE: do not modify the tests to allow alternative inputs and make sure the tests are passing.
```

Comment:
Is this something we can detect? It is perhaps not very user-friendly to display this message if the user didn't actually modify the tests.

Reply:
Since the disapprovals are passed to a mentor, you don't really need to guard for that. I think you _may_ assume the tests are unaltered, for now.